webpack.config.electron.js

```diff
@@ -8,31 +12,47 @@ export default {
 
   entry: './main.development',
 
+  // 'main.js' in root
   output: {
     ...baseConfig.output,
     path: __dirname,
     filename: './main.js'
   },
 
   plugins: [
+    // Minify the output
     new webpack.optimize.UglifyJsPlugin({
       compressor: {
         warnings: false
       }
     }),
+
+    // Add source map support for stack traces in node
+    // https://github.com/evanw/node-source-map-support
     new webpack.BannerPlugin(
       'require("source-map-support").install();',
       { raw: true, entryOnly: false }
     ),
+
+    // NODE_ENV should be production so that modules do not perform certain development checks
     new webpack.DefinePlugin({
       'process.env': {
```

Comment:
Good time to make this match `webpack.config.development.js`

`'process.env.NODE_ENV': JSON.stringify('production')`